src/index.html

```diff
@@ -1,14 +1,58 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="src/images">
+        <img class="logo" src="images/logo.png"
+        alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav_item">
+            <a href="text" class="is-active blue_text nav__link">
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="text" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="text" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a href="text" class="nav__link change_color"
+            data-qa="hover">Laptops & Computers</a>
```

Comment:
redundant class and code styling
```suggestion
            <a 
              href="text" 
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

Reply:
done